CMakeLists.txt

```diff
@@ -248,6 +245,11 @@ write_basic_package_version_file(
   COMPATIBILITY AnyNewerVersion
 )
 
+set(REF_BLAS_LIBNAME ${BLAS_file})
+
+# Add source directory and output to bin/
+add_subdirectory(src bin)
```

Comment:
Is there a reason why this line was moved below?

Reply:
This line was moved because add_subdirectory was calling the cmake `configure_file()` to create config.hpp before FindCBLAS.cmake was setting the BLAS_file variable.

Reply:
okay, thanks!